bin/rebuild

```diff
@@ -31,6 +32,38 @@ versiondb_head() {
   (cd "$VERSIONDB" && git rev-parse HEAD)
 }
 
+save_build_env() {
+  # saving environment information
+  filename="${LSSTSW_BUILD_DIR}/${BUILD}.env"
+  echo "Saving environment information in ${filename}"
+  conda env export | grep -v "^name"> /tmp/lenv
```

Comment:
I think use of the `diff -I` option might avoid a lot of this file manipulation.

Reply:
So far I prefer to keep it as is.